taca/storage/storage.py

```diff
@@ -64,6 +81,93 @@ def archive_to_swestore(days, run=None):
                 else:
                     LOG.info('No old runs to be archived')
 
+
+def cleanup_swestore(days,dry_run=False):
+    """
+        Remove archived runs from swestore
+
+        :param int days: Threshold days to check and remove
+    """
+    config = get_config()
+    LOG = get_logger()
+    runs = filesystem.list_runs_in_swestore(path=config.get('cleanup').get('swestore'))
+    for run in runs:
+        date = run.split('_')[0]
+        if misc.days_old(date) > days:
+            if dry_run:
+                LOG.info('Will remove file {} from swestore'.format(run))
+                continue
+            misc.call_external_command('irm -f {}'.format(run))
+            LOG.info('Removed file {} from swestore'.format(run))
+
+def cleanup_uppmax(site,days,dry_run=False):
+    """
+        Remove project/run that have been closed more than 'days'
+        from the given 'site' on uppmax
+
+        :param str site: site where the cleanup should be performed
+        :param int days: number of days to check for closed projects
+    """
+    config = get_config()
+    LOG = get_logger()
+    root_dir = config.get('cleanup').get(site)
+    deleted_log = config.get('cleanup').get('deleted_log')
+    assert os.path.exists(os.path.join(root_dir,deleted_log)), "Log directory {} doesn't exist in {}".format(deleted_log,root_dir)
+    log_file = os.path.join(root_dir,"{fl}/{fl}.log".format(fl=deleted_log))
+    PRO_RE = '[a-zA-Z]+\.[a-zA-Z]+_\d{2}_\d{2}'
+    db_config = config.get('statusdb',{})
+
+    # make a connection for project db #
+    pcon = statusdb.ProjectSummaryConnection(**db_config)
```

Comment:
If the `~/.ngi_config/statusdb` is required in `statusdb` package, go for the second one. Otherwise the first one, specifying all parameters I think is still better than `**db_config` :-)